core/src/main/java/com/linecorp/armeria/server/HttpServerPipelineConfigurator.java

```diff
@@ -448,20 +447,38 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
 
         private final ServerHttp1ObjectEncoder responseEncoder;
         @Nullable
+        private final KeepAliveHandler keepAliveHandler;
+        @Nullable
         private String name;
 
         Http2PrefaceOrHttpHandler(ServerHttp1ObjectEncoder responseEncoder) {
             this.responseEncoder = responseEncoder;
+            keepAliveHandler = responseEncoder.keepAliveHandler();
```

Comment:
Question: do we need this handler for `Http2PrefaceOrHttpHandler`?

Reply:
Yes. when a client connects a connection and does not send any request, we have to close the connection after an idle timeout.

Reply:
The following test failed without this. 😀
https://github.com/line/armeria/blob/46eff82ddbf52af751fb31debc2273638ae6a5f9/core/src/test/java/com/linecorp/armeria/server/ServerTest.java#L286